docs/ibis-for-pandas-users.ipynb

```diff
@@ -31,6 +31,7 @@
    ]
```

Comment:
<p>It's not obvious in the reviewnb UI, but I changed the header level on some of these headings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6447/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>